frontend/src/pages/detail/PODetail.tsx

```diff
@@ -142,18 +134,18 @@ export default function PODetail() {
       filterPlaceholder: "Price",
       cellEditValidator: (event: ColumnEvent) => event.newValue > 0,
       cellEditor: (options: ColumnEditorOptions) => priceEditor(options),
+      customBody: (rowData: POPurchaseRow) =>
+        priceBodyTemplate(rowData.unitWholesalePrice),
     },
     {
       field: "subtotal",
       header: "Subtotal ($)",
       filterPlaceholder: "Subtotal",
+      customBody: (rowData: POPurchaseRow) =>
+        priceBodyTemplate(rowData.subtotal),
     },
   ];
 
-  const onCellEditComplete = (event: ColumnEvent) => {
```

Comment:
You remove this function here, and it isn't reintegrated anywhere else.

Reply:
I actually include it in Table.tsx (now TableColumns.tsx). The reason of this is that for any column that is currently editable, the only thing we ever want to do is set the field to be equal to the new value. Therefore, rather than requiring columns to define this function, I just made this the default for all columns, which only takes action if the column is editable